src/applications/hca/config/form.js

```diff
@@ -520,6 +557,47 @@ const formConfig = {
             },
           },
         },
+        documentUpload: {
+          title: 'Upload your discharge papers',
+          path: 'military-service/documents',
+          depends: formData =>
+            !formData['view:isUserInMVI'] && !environment.isProduction(),
+          editModeOnReviewPage: true,
+          uiSchema: {
+            'ui:title': 'Upload your discharge papers',
+            'ui:description': fileHelp,
+            attachments: fileUploadUI('', {
+              buttonText: 'Upload a document',
+              addAnotherLabel: 'Upload another document',
+              fileUploadUrl: `${environment.API_URL}/v0/hca_attachments`,
+              fileTypes: ['pdf', 'doc', 'docx', 'jpg', 'jpeg', 'rtf', 'png'],
+              maxSize: 1024 * 1024 * 10,
```

Comment:
The description suggests this is a cumulative limit. I think maxSize applies to the single file. Do we know what happens when they upload more than 10MB of files? 

See: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14122

Reply:
good catch. yeah this is a known issue that we aren't worried about at this point since it's such an edge case. but sadly at this point there is no way to show errors when the _total_ upload size is too large. and if someone does upload multiple files that exceed the backend's limit (which is actually closer to 15MB) then the submission will fail and it will be a horrible UX :(

If I made a ticket to enhance the file upload ui widget to add this support should I tag your team on that? I'm not sure who would be responsible for an enhancement like that.

Reply:
yeah you can tag our team since it's a `forms-system` enhancement. this will be the second form I'm aware of that needs a cumulative size limit enforced.